src/images/google-svg/LICENSE.md

```diff
@@ -0,0 +1,14 @@
+The files in this directory were generated on 2/25/2025 using Google's Icon generator.
+
+At the time the icons were generated, they were subject to the APACHE LICENSE, VERSION 2.0:
+https://www.apache.org/licenses/LICENSE-2.0.html
+
+URLs to regenerate the icons:
```

Comment:
Is it worth listing each link separately? Also, the names here don't match the filenames for the icons, should they?

You could use relative links to the svg files in the source tree to embed the images in the markdown, to show what each link renders as.

Reply:
I think this is just temporary.  I'd really like to get us to the font based version of this.
More than anything, I wanted to be clear on where the files in this directory are coming from and how to regenerate (most of) them, if needed.